realm/realm-library/src/main/java/io/realm/BaseRealm.java

```diff
@@ -45,7 +46,7 @@
  * @see io.realm.Realm
  * @see io.realm.DynamicRealm
  */
-abstract class BaseRealm implements Closeable {
+public abstract class BaseRealm implements Closeable {
```

Comment:
`BaseRealm` is used in the `RealmObjectProxy` interface. It's needed for `DynamicRealmObject` that works with with a `DynamicRealm` instead of `Realm` hence the use of the super type `BaseRealm` to share implementation